modules/odc-uninstalling-web-terminal.adoc

```diff
@@ -84,6 +84,28 @@ $ oc delete customresourcedefinitions.apiextensions.k8s.io devworkspaceroutings.
 $ oc delete customresourcedefinitions.apiextensions.k8s.io devworkspaces.workspace.devfile.io
 ----
 +
+[source,terminal]
+----
+$ oc delete customresourcedefinitions.apiextensions.k8s.io devworkspacetemplates.workspace.devfile.io
+----
++
+[source,terminal]
+----
+$ oc delete customresourcedefinitions.apiextensions.k8s.io devworkspaceoperatorconfigs.controller.devfile.io
+----
++
+[WARNING]
+====
+The DevWorkspace Operator ships custom resource definitions that make use of conversion webhooks. Failing to remove the CRDs above can cause issues on the cluster, as the steps below will remove the server that handles these conversion webhooks.
```

Comment:
**Rephrase suggestion:** The DevWorkspace Operator transmits custom resource definitions (CRDs) that use conversion webhooks. Failing to remove the CRDs can cause issues on the cluster. The following steps will remove the server that handles these conversion webhooks.

Reply:
I would suggest `provides` instead of `transmits`. Rest of the suggested change looks good.

Reply:
:+1: